docs/sdk-and-tools/indices/transactions.md

```diff
@@ -0,0 +1,81 @@
+---
+id: transactions
```

Comment:
as discussed, please set the `id` to `es-index-transactions`

Reply:
done